src/platform/mhv/secondary-nav/tests/MhvSecondaryNavItem.unit.spec.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import MhvSecondaryNavItem from '../components/MhvSecondaryNavItem';
+
+describe('MHV Secondary Navigation Item Component', () => {
+  describe('handle abbreviations', () => {
+    it('when provided', () => {
+      const title = 'a title';
+      const abbr = 'an abbr';
+      const { getByRole } = render(
+        <MhvSecondaryNavItem
+          title={title}
+          abbreviation={abbr}
+          iconClass="fas fa-home"
+          href="/my-health"
+        />,
+      );
+      const link = getByRole('link');
+      expect(link.text.match(new RegExp(title, 'g'))?.length).to.eql(1);
+      expect(link.text.match(new RegExp(abbr, 'g'))?.length).to.eql(1);
```

Comment:
It isn't 100% clear to me what is being tested here since if an abbreviation is provided, it renders: `<abbr title={title}>{abbreviation}</abbr>`

Reply:
I changed it up to use getByText instead. Should be clearer now.